packages/core/src/external/carequality/ihe-gateway-v2/outbound/ihe-gateway-v2-logic.ts

```diff
@@ -151,19 +161,35 @@ export async function createSignSendProcessXCPDRequest({
       cxId,
       index,
     });
-    try {
-      // TODO not sure if we should retry on timeout
-      await executeWithNetworkRetries(async () => axios.post(pdResponseUrl, result), {
-        initialDelay: 100,
-        maxAttempts: 5,
-        httpStatusCodesToRetry: [502, 504],
-        log,
-      });
-    } catch (error) {
-      const msg = "Failed to send PD response to internal CQ endpoint";
-      const extra = { cxId, patientId, result };
-      log(`${msg} - ${errorToString(error)} - ${JSON.stringify(extra)}`);
-      capture.error(msg, { extra: { ...extra, error } });
+    if (result.patientMatch) {
+      try {
+        // TODO not sure if we should retry on timeout
+        await executeWithNetworkRetries(async () => axios.post(pdResponseUrl, result), {
+          initialDelay: 100,
+          maxAttempts: 5,
+          httpStatusCodesToRetry: [502, 504],
+          log,
+        });
+      } catch (error) {
+        const msg = "Failed to send PD response to internal CQ endpoint";
+        const extra = { cxId, patientId, result };
+        log(`${msg} - ${errorToString(error)} - ${JSON.stringify(extra)}`);
+        capture.error(msg, { extra: { ...extra, error } });
+      }
+    } else {
+      try {
+        const filePath = `cxId=${cxId}/patientId=${patientId}/requestId=${result.id}/gwId=${result.gateway.id}/result.json`;
+        await s3Utils.uploadFile({
+          bucket,
+          key: filePath,
+          file: Buffer.from(JSON.stringify(result), "utf8"),
+        });
```

Comment:
I think we should also set the `contentType`.

Reply:
Done.